store/rootmulti/store.go

```diff
@@ -230,6 +230,10 @@ func (rs *Store) loadVersion(ver int64, upgrades *types.StoreUpgrades) error {
 	for key := range rs.storesParams {
 		storesKeys = append(storesKeys, key)
 	}
+	sort.Slice(storesKeys, func(i, j int) bool {
```

Comment:
It's sorted in following code, and it don't affect the order in commit, to enforce order in commit, we need to sort there j think.

Reply:
yes, you're right